### ISSUE TITLE
IndexError: list index out of range error in pandoc-latex-environment 1.1.6.2

### ISSUE THREAD

**Issue by cryptax** [chdemko/pandoc-latex-environment#15]
I get this error when using pandoc-latex-environment. Do you have an idea what's the problem?

I have:

- pip3
- pandox-latex-environment 1.1.6.2
- python 3.10.2
- linux
- pyenv

```
Traceback (most recent call last):
  File "/home/axelle/.pyenv/versions/3.10.12/bin/pandoc-latex-environment", line 8, in <module>
    sys.exit(main())
  File "/home/axelle/.pyenv/versions/3.10.12/lib/python3.10/site-packages/pandoc_latex_environment.py", line 114, in main
    return run_filter(transform, doc=doc, prepare=prepare)
  File "/home/axelle/.pyenv/versions/3.10.12/lib/python3.10/site-packages/panflute/io.py", line 227, in run_filter
    return run_filters([action], *args, **kwargs)
  File "/home/axelle/.pyenv/versions/3.10.12/lib/python3.10/site-packages/panflute/io.py", line 200, in run_filters
    doc = load(input_stream=input_stream)
  File "/home/axelle/.pyenv/versions/3.10.12/lib/python3.10/site-packages/panflute/io.py", line 58, in load
    doc = json.load(input_stream, object_hook=from_json)
  File "/home/axelle/.pyenv/versions/3.10.12/lib/python3.10/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/home/axelle/.pyenv/versions/3.10.12/lib/python3.10/json/__init__.py", line 359, in loads
    return cls(**kw).decode(s)
  File "/home/axelle/.pyenv/versions/3.10.12/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/home/axelle/.pyenv/versions/3.10.12/lib/python3.10/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
  File "/home/axelle/.pyenv/versions/3.10.12/lib/python3.10/site-packages/panflute/elements.py", line 1431, in from_json
    return _res_func[tag](c)
  File "/home/axelle/.pyenv/versions/3.10.12/lib/python3.10/site-packages/panflute/table_elements.py", line 396, in table_from_json
    ica = decode_ica(c[0])
  File "/home/axelle/.pyenv/versions/3.10.12/lib/python3.10/site-packages/panflute/utils.py", line 21, in decode_ica
    return {'identifier': lst[0],
IndexError: list index out of range
Error running filter pandoc-latex-environment:
Filter returned error status 1
make: *** [Makefile:13: sponsorship.pdf] Error 83
```

**Comment by cryptax** [comment]
The issue does not occur with **Python 3.8.17**, so I suppose pandoc-latex-environment does not support Python 3.10?

**Comment by chdemko** [comment]
pandoc-latex-environment uses last version of panflute which is responsable for this error. What is the pandoc version used ?

**Comment by redds-be** [comment]
I had the exact same problem, using pandoc-latex-environment 1.1.6.2 and Python 3.10.
The issue has been fixed by updating to pandoc 3.1.9, there is nothing wrong (at least related to the issue) with pandoc-latex-environment, that being said, I suppose this issue can now be closed.
